### ISSUE TITLE
Node 1.10 stop with error

### ISSUE THREAD

**Issue by Sebdens** [Shardeum/shardeum-bug-reporting#538]
Hello.

My node was validating and stopped suddenly with the above message. Can't restart it

The code shown when I try to restart it:

"Node exited with the following message:
invoke-exit: node restarted ungracefully, needs to restart exitUncleanly"

![Capture d’écran (215)](https://github.com/shardeum/shardeum-bug-reporting/assets/120269832/79c03941-1fcc-4e5a-ac1b-908bb9aa2ec3)

**Comment by airdrop4cb** [comment]
I got the same error as well, and I couldn't resolve this issue. I had to re-create the image and restart the node. Now 1) I lost the funds from the staking address wallet as I staked the funds with previous validator, and unable to unstake with new one.. 2) new validator I am unable to get inot network.. it's always in standby state. 

staking address: 0x1aFFC83874BC5E1ba75F6c746e231a8092541f13 for reference. created ticket for the same yesterday.

**Comment by Sebdens** [comment]
I haven't touched the node since yesterday and it restarted on its own. It is currently in standby mode and seems to work normally...

**Comment by airdrop4cb** [comment]
That’s good it resolved the error by itself. It wasn’t the case for me. However you lost the spot of active validators I guess. I think we get rewarded only if it is active( dashboard says). I guess only 1000 validators active at anytime. Network tab on dashboard says always 1000 or 1002 active validators always.

**Comment by Sebdens** [comment]
Yes I lost my place as an active validator but it runs regularly and I don't know if it stopped during or at the end of my validation period. The main thing is that I did not try to reinstall the node and therefore I did not lose my rewards which are due to the active validation period. I hope yours will restart on its own too ;)
